### PR TITLE
add method cards to examples site and remove TTS handbook

### PR DESCRIPTION
**Pull request by wslack**
Per https://github.com/18F/federalist-docs/issues/90

### CODE REVIEW

File: pages/about-federalist/example-sites.md [hunk 1]
```diff
@@ -26,8 +23,12 @@ All of these sites are currently hosted on Federalist. Their source code is also
 **[Office of Evaluative Sciences](https://oes.gsa.gov/)**
 ![Office of Evaluative Sciences]({{site.baseurl}}/uploads/partner-sites/oes-gsa-gov.png)
 
+
+**[18F Method Cards](https://methods.18f.gov/)**
+![18F Method Cards]({{site.baseurl}}/uploads/partner-sites/method-cards.png)
+
 **[Tech at GSA](https://tech.gsa.gov/)**
-![Tech at GSA]({{site.baseurl}}/uploads/partner-sites/tech-gsa-gov.png)
+![Tech at GSA page by GSA IT]({{site.baseurl}}/uploads/partner-sites/tech-gsa-gov.png)
```
Comment (jseppi) [inline pages/about-federalist/example-sites.md]: Did you also want to change the "label" for this image to `[Tech at GSA page by GSA IT]`? This change just modifies the image's `alt` text so it isn't really visible to anyone.

Comment (wslack) [inline pages/about-federalist/example-sites.md]: yeah good point

Comment (wslack) [inline pages/about-federalist/example-sites.md]: fixed!

File: pages/about-federalist/example-sites.md [hunk 2]
```diff
@@ -26,8 +23,11 @@ All of these sites are currently hosted on Federalist. Their source code is also
 **[Office of Evaluative Sciences](https://oes.gsa.gov/)**
 ![Office of Evaluative Sciences]({{site.baseurl}}/uploads/partner-sites/oes-gsa-gov.png)
 
-**[Tech at GSA](https://tech.gsa.gov/)**
-![Tech at GSA]({{site.baseurl}}/uploads/partner-sites/tech-gsa-gov.png)
+**[18F Method Cards](https://methods.18f.gov/)**
+![18F Method Cards]({{site.baseurl}}/uploads/partner-sites/method-cards.png)
+
+**[Tech at GSA (GSA IT)](https://tech.gsa.gov/)**
+![Tech at GSA page(GSA IT)]({{site.baseurl}}/uploads/partner-sites/tech-gsa-gov.png)
```
Comment (jseppi) [inline pages/about-federalist/example-sites.md]: add space between `page` and `(GSA IT)`

Comment (wslack) [inline pages/about-federalist/example-sites.md]: fixed!

**Comment by jseppi** [comment]
👍 LGTM

**Pull request merged**